cumulusci/tasks/bulkdata/load.py

```diff
@@ -86,19 +79,21 @@ def _run_task(self):
             if not started and start_step and name != start_step:
                 self.logger.info(f"Skipping step: {name}")
                 continue
+
             started = True
 
-            self.logger.info(f"Running Job: {name}")
+            self.logger.info(f"Running step: {name}")
             result = self._load_mapping(mapping)
-            if result not in ("Completed", "CompletedWithFailures"):
-                raise BulkDataException(f"Job {name} did not complete successfully")
+            if result is Status.FAILURE:
```

Comment:
Are you intentionally switching to always considering row-level errors as a failure of the entire task regardless of the ignore_row_errors option?

Reply:
Nope. I missed this in doing a merge. Having different error semantics here and in the mixin is confusing; I will rework it to use an enum throughout and consistent logic.